### ISSUE TITLE
Tests - ON not including pending tests

### ISSUE THREAD

**Issue by johanley** [ccodwg/Covid19Canada#28]
Although I don't see it stated anywhere, you seem to be reporting on the total number of tests performed, including pending tests.

If that's the case, your reporting for Ontario seems to be off. For example, yesterday, on April 22, ON reported these numbers:

- Total tests completed 184,531
- Currently under investigation 6,845  (Samples with testing in progress.)

Your reported testing number for ON for that day is 184,531. If you are indeed reporting completed+pending, shouldn't you be reporting the sum of the above two numbers?

**Comment by jeanpaulrsoucy** [comment]
Hi John,

Thanks for flagging this. I took a moment to look into this - our original intention is indeed to include the total number of tests done.

When Ontario switched to their new website at the end of March, they initially stopped reporting the number of tests that were pending - they reported only "Total tested".

https://web.archive.org/web/20200330021143/https://www.ontario.ca/page/2019-novel-coronavirus

By March 31 (the beginning of your archive) this was changed to "Total tested" and "Currently under investigation" (not entirely clear if the former includes the latter). This has since been changed to "Total tests completed" and "Currently under investigation", which has removed the ambiguity.

Of course, Ontario recently started reporting samples taken rather than patients tested, which is partly responsible for the sharp spike in daily numbers from Ontario in the past week. We should probably add a methodology note to explicitly address the variation in the definition of this quantity across time and geography.

Regarding the Ontario testing numbers, we will make a determination whether to go back and update them to include total + pending.

Jean-Paul

**Comment by jeanpaulrsoucy** [comment]
Hi John,

Thanks for flagging this. I took a moment to look into this - our original intention is indeed to include the total number of tests done.

Originally, Ontario reported "Total number of patients approved for COVID-19 testing to date", which included negative, positive, and under investigation. Near the end of March, they briefly stopped reporting the number of tests that were pending - they reported only "Total tested".

https://web.archive.org/web/20200330171616/https://www.ontario.ca/page/2019-novel-coronavirus

By March 31 (the beginning of your archive) this was changed to "Total tested" and "Currently under investigation" (not entirely clear if the former includes the latter, as it did under the previous wording). This has since been changed to "Total tests completed" and "Currently under investigation", which has removed the ambiguity - the total tested number no longer includes those under investigation.

On a related note, Ontario recently started reporting samples taken rather than patients tested, which is partly responsible for the sharp spike in daily numbers from Ontario in the past week. We should probably add a methodology note to explicitly address the variation in the definition of this quantity across time and geography.

Regarding the Ontario testing numbers, I will make a point to go back and update the numbers. We can use your archive (easier than Wayback Machine), since it begins almost precisely after the apparently change in reporting.

Thanks again.

Jean-Paul
